src/electionguard-ui/ElectionGuard.UI.Lib/Services/Database/DbService.cs

```diff
@@ -46,6 +53,17 @@ public static class DbService
     private static string DbConnection = string.Empty;
     private static MongoClient client = new();
 
+    public static event EventHandler<DbEventArgs> DatabaseDisconnected;

```

Comment:
Interesting pattern. I kind of like this a little better than explicit delegates